src/Reader.js

```diff
@@ -53,6 +43,41 @@ Reader.open = function(file, cb){
 
 Reader.openSync = Reader;
 
+Reader.prototype.setup = function(){
+  var metaPosition = this.findMetaPosition();
+  this.metadata = this.readData(metaPosition).value;
+
+  this.setRecordSize(this.metadata.record_size);
+  this.recordSize = this.metadata.record_size;
+  this.nodeSize = this.recordSize / 4;
+  this.numNodes = this.metadata.node_count;
+  this.dataStart = this.numNodes * this.nodeSize + 16;
+  this.ip = new IpDecoder();
+  this.findIPv4StartPointer();
+};
+
+Reader.prototype.reload = function(file, cb){
```

Comment:
could we make it so `reader.reload(cb)` is able to just re-read the file that was originally passed in? not a major issue but just a little prettier if you're simply replacing the file